go.mod

```diff
@@ -29,3 +28,5 @@ require (
 )
 
 replace github.com/spf13/cobra => github.com/gapra-msft/cobra v1.4.1-0.20220411185530-5b83e8ba06dd
+
+replace github.com/sevlyar/go-daemon => github.com/cvvz/go-daemon v0.1.5
```

Comment:
Why we are doing this ? Is there something we have modified in the go-daemon itself ?

Reply:
This is not safe if we are modifying the third-party tool. Maintenance is a big question here and on top of that we cannot merge back with any critical fixes that comes in the main repo. Lot of manual effort will be there to keep these in sync, and we do not want to take such dependencies.

Reply:
I've filed a [pr ](https://github.com/sevlyar/go-daemon/pull/90)to go-daemon

Reply:
Let me take up this PR from here. I get what you are trying to achieve, let me fix it as per our regular blobfuse stnadards.

Reply:
Can you check the changes now

Reply:
I've left some comments, the other looks good to me.